content/en/user-manual/scripting/migration-guide.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Migration Guide
+template: usermanual-page.tmpl.html
+position: 10
+---
+
+## Migrating Legacy Script Projects
+
+In July 2016, PlayCanvas adopted its [current scripting system][1] (sometimes known as Scripts 2.0). Legacy script projects continue to work as normal. However, it is no longer possible to fork or create new legacy script projects.
+
+In December 2020, it was announced that legacy script projects would be made read-only in the near future. When you open one in the Editor, you will see the following message in the project dashboard:
+
+![Dashboard Legacy Script][2]
+
+And the following message in the Editor:
+
+![Editor Legacy Script][3]
+
+If you want to continue to work on legacy script projects, we recommend that you migrate them to the current format. Unfortunately, there is no automated migration process for this. Instead, it is a manual process, but this guide will walk you through it.
+
+### Step 1 - Create a New Project
+
+We will begin by creating a new, blank project to receive the update project. Delete the four entities created by default below the scene root entity.
```

Comment:
"We will begin by creating a new, blank project to transfer assets and code into."

Reply:
Done.